serverless/admin_guide/serverless-ha.adoc

```diff
@@ -6,14 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-High availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
-In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+High availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
 
-HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving or Eventing control plane is installed.
-
-When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
-These controller instances compete to use a shared resource, known as the leader election lock.
-The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving or Eventing control plane is installed. When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
```

Comment:
Passive voice:

is available,
is enabled,
is installed,
are scheduled,
are required